src/UI/Internal/RenderConfig.elm

```diff
@@ -0,0 +1,39 @@
+module UI.Internal.RenderConfig exposing (Locale(..), RenderConfig(..), RenderConfigData, string, strings)
+
+import Element
+import I18n.English as LangEnglish
+import I18n.Spanish as LangSpanish
+import I18n.Type as I18n
+
+
+type alias RenderConfigData =
+    { deviceClass : Element.DeviceClass
+    , deviceOrientation : Element.Orientation
+    , locale : Locale
+    }
+
+
+{-| `RenderConfig.RenderConfig` upholds all the information required for the components to apply the responsivity and accessibility changes.
+-}
+type RenderConfig
+    = RenderConfig RenderConfigData
+
+
+type Locale
+    = EnglishGB
+    | SpanishES
+
+
+string : (I18n.Root -> String) -> RenderConfig -> String
```

Comment:
Is this function used?

Reply:
No, this was a proposal of another syntax I intended to do for you to review, but it seems worst:
`(RenderConfig.string (.fields >> .range >> .to) renderConfig)`
instead of
`(RenderConfig.strings renderConfig).fields.range.to`

WDYT?